code/datums/uplink_item.dm

```diff
@@ -324,15 +327,12 @@ var/list/uplink_items = list()
 	desc = "A fully-loaded Donksoft bullpup submachine gun that fires riot grade rounds with a 20-round magazine."
 	item = /obj/item/weapon/gun/projectile/automatic/c20r/toy
 	cost = 8
-	gamemodes = list(/datum/game_mode/nuclear)
```

Comment:
im assuming this means it's not restricted solely to nuke op rounds, but then why is the surplus removed on this and not on the next one
what does surplus even mean
my puny non-coder brain is confused